sormas-api/src/main/resources/strings.properties

```diff
@@ -479,7 +479,7 @@ infoSelectOrCreateContactImport = The database already contains at least one con
 infoSelectOrCreatePersonForContact = The database already contains at least one person that seems to be very similar to the personal details of the created contact.<br/><br/>Please look through the list of persons. If you feel certain that one of those persons matches your contact person, select it and click on the <i>Save</i> button. Otherwise, click on <i>Create New Person</i> to create a new person for your contact.<br/><br/>If you are unsure, you can discard this window and cancel the contact creation process.
 infoSelectOrCreatePersonForContactImport = The database already contains at least one person that seems to be very similar to the personal details of the imported contact.<br/><br/>Please look through the list of persons. If you feel certain that one of those persons matches your contact person, select it and click on the <i>Save</i> button. Otherwise, click on <i>Create New Person</i> to create a new person for your contact.<br/><br/>If you are unsure, you can discard this window to remove the contact from the current import.
 infoSelectOrCreatePersonForEventParticipant = The database already contains at least one person that seems to be very similar to the personal details of the created event participant.<br/><br/>Please look through the list of persons. If you feel certain that one of those persons matches your event person, select it and click on the <i>Save</i> button. Otherwise, click on <i>Create New Person</i> to create a new event person for your event.<br/><br/>If you are unsure, you can discard this window and cancel the event participant creation process.
-infoPickOrCreateEventForCase = The list below contains all existing events having the same disease as the current case. Search for the event first in case not found create a new one.
+infoPickOrCreateEventForCase = Please check whether the event this case belongs to is already on this list or create a new one if it isn't.
```

Comment:
Ah, sorry, the first part was actually fine! I just wanted you to replace the second sentence because it was a bit unclear for me. Re-add the first sentence again, please :)